net-analyzer/nmap/nmap-3.81.recipe

```diff
@@ -56,8 +62,9 @@ BUILD()
 	libtoolize --force --copy --install
 	popd
 
-	runConfigure --omit-dirs "dataRootDir docDir" \
-		./configure  --without-nmapfe LDFLAGS=-lnetwork
+	runConfigure --omit-dirs "binDir dataRootDir docDir" \
+		./configure --bindir="$prefix/bin" \
```

Comment:
Same here for $commandBinDir: https://github.com/Begasus/haikuports/wiki/Useful-variables-for-recipes#commandbindir-and-commandsuffix

Reply:
even better there should be a similar helper for CMake as well (which needs some rework anyway as currently `cmakeDirArgs` are all absolute which does not work so well on some recipes)

Reply:
This should be dealth with at haikuporter I guess, @davidkaroly good to go?

Reply:
i think so